### PR TITLE
Documentation updates for the initial release

### PR DESCRIPTION
**Pull request by yspolyakov**

### CODE REVIEW

File: docs/sphinx_rsts/intro/using_the_library.rst [hunk 1]
```diff
@@ -7,33 +7,19 @@ The source files in the demo directories, particularly those in src/examples/pke
...
-
   * ``BFVrns``
 
-  * ``BFVrnsB``
-
   * ``BGVrns``
 
-  * ``CKKS``
-
-  * ``Null`` (a test scheme which does not actually encrypt/decrypt anything)
```
Comment (iquah1) [inline docs/sphinx_rsts/intro/using_the_library.rst:30]: Did we remove the `Null` scheme? Is there a reason why we did it?

**Review by iquah1 — COMMENTED** [review]
Just one question but looks good to me. Let's merge after looking at the output and verifying everything, but so far I don't think that it will break any formatting

File: docs/sphinx_rsts/intro/using_the_library.rst (hunk 1 above)
Comment (yspolyakov) [reply docs/sphinx_rsts/intro/using_the_library.rst:30]: yes, we removed it. The previous null scheme was only for BGV and BFV. It was more confusing to keep it.

**Review by iquah1 — APPROVED** [review]
LGTM

**Pull request merged**